src/ItemFlag/FlagItemButton.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { FC } from 'react';
+import { useTranslation } from 'react-i18next';
+import { IconButton, Tooltip } from '@material-ui/core';
+import ReportIcon from '@material-ui/icons/Report';
+
+export interface FlagItemButtonProps {
+  setOpen: Function;
+}
+
+export const FlagItemButton: FC<FlagItemButtonProps> = ({ setOpen }) => {
+  const { t } = useTranslation();
+
+  const openItemFlagDialog = (): void => {
+    setOpen(true);
+  };
+
+  return (
+    <Tooltip title={t('Report')}>
+      <IconButton color='primary' onClick={openItemFlagDialog}>
+        <ReportIcon fontSize='large' />
```

Comment:
Are other buttons using `large`? I would add it in the props, and set a default value. Same for `primary`.